src/remotestorage.js

```diff
@@ -682,7 +667,16 @@ RemoteStorage.prototype = {
   },
 
   /**
-   * TODO: document
+   * Start synchronization with remote storage, downloading and uploading any
+   * changes within the cached paths.
+   *
+   * Please consider: local changes will attempt sync immediately, and remote
+   * changes should also be synced timely when using library defaults. So
+   * this is mostly useful for letting users sync manually, when pressing a
+   * sync button for example. This might feel safer to them sometimes, esp.
+   * when shifting between offline and online a lot.
+   *
+   * TODO: move to sync.js
```

Comment:
What about the TODO there? Didn't you just move the doc?

Reply:
I moved this comment that was prefixed with `startSync` from the top of the file to the top of the `startSync` function. Is `TODO: move to sync.js` saying that the comment needs to be moved to the `sync.sync` function?

Reply:
I'm not sure what's unclear about the language there. But I don't see why it should be moved. I asked you what's with the TODO there, because you looked at the doc and apparently already moved it to the appropriate place.